lib/taskana-core/src/test/java/acceptance/workbasket/DeleteWorkbasketAccTest.java

```diff
@@ -0,0 +1,91 @@
+package acceptance.workbasket;
```

Comment:
Bitte noch einen Test zum Löschen des WBs als Distribution-Ziel in einem anderen WB ergänzen.

Reply:
done